packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -330,6 +330,27 @@ class Editor implements KetcherEditor {
     return this.render.options.zoom
   }
 
+  zoomAccordingContent() {
```

Comment:
Just curious, there is "autoScale" property in the render options, that do something similar, can't use use it?

Reply:
Looks like autoscale is about scaling the size of structures after its addition on canvas:
https://github.com/epam/ketcher/assets/14859362/cd6c1f72-ab56-4caf-b032-921fcc89e7fd